### ISSUE TITLE
time_offset v4 option is wrong signedness 

### ISSUE THREAD

**Issue by ColinMcInnes** [NetworkConfiguration/dhcpcd#318]
dhcpcd/src/dhcpcd-definitions.conf

time_offset (dhcp option 2) is incorrectly trying to encode/parse as uint, it
needs to be int so we can do western timezone offsets (negative).

```
##############################################################################
# DHCP RFC2132 options unless otheriwse stated
define 1	request ipaddress	subnet_mask
# RFC3442 states that the CSR has to come before all other routes
# For completeness we also specify static routes then routers
define 121	rfc3442			classless_static_routes
define 2	uint32			time_offset                          <<<< should be int as per RFC-2132
define 3	request array ipaddress	routers
define 4	array ipaddress		time_servers 
define 5	array ipaddress		ien116_name_servers
define 6	array ipaddress		domain_name_servers

```

**Comment by ColinMcInnes** [comment]
Resolved in Pull Request #319

**Comment by rsmarples** [comment]
Thanks for the patch!
